src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -78,6 +79,15 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('output_index', help='The element index of the output to remove.',
                    type=int, default=None)
 
+    with self.argument_context('ams transform output add') as c:
+        c.argument('preset', help='Allowed values: {}. In addition to the allowed values, you can also pass the local full path to a custom preset JSON file.'
+                   .format(", ".join(get_presets_definition_name_completion_list())))
+        c.argument('audio_insights_only', arg_group='Video Analyzer', action='store_true', help='Use this flag to only extract audio insights when processing a video file.')
+        c.argument('audio_language', arg_group='Audio/Video Analyzer', help='The language for the audio payload in the input using the BCP-47 format of \"language tag-region\" (e.g: en-US). The list of supported languages are: {}.'
```

Comment:
You should just show Allowed values: {values}, following how an enum is shown. 

So that The list of supported languages are would be replaced for Allowed values

Reply:
agree.. the pattern is `Allowed values: {values}`